assets/scss/frontend/theme-compatibility.scss

```diff
@@ -36,4 +36,20 @@ body.give-twentysixteen {
     padding-left: 0;
   }
 
-}
\ No newline at end of file
+}
+
+//Twenty Seventeen Dark Color Scheme
+body.give-twentyseventeen.colors-dark {
```

Comment:
@DevinWalker Ya, those were supposed to be the only real changes... not sure how the inline comments got formatted differently at all. And not totally sure how to resolve that since I don't see actual differences above... maybe a tabs versus spaces issue!?